custom_components/zwave_mqtt/services.py

```diff
@@ -0,0 +1,130 @@
+"""Methods and classes related to executing Z-Wave commands and publishing these to hass."""
+import logging
+import voluptuous as vol
+import homeassistant.helpers.config_validation as cv
+from homeassistant.core import callback
+
+from . import const
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class ZWaveServices:
+    """Class that holds our services ( Zwave Commands) that should be published to hass."""
+
+    def __init__(self, hass, manager):
+        """Initialize with both hass and ozwmanager objects."""
+        self._hass = hass
+        self._manager = manager
+
+    @callback
+    def register(self):
+        """Register all our services."""
+        instance_schema = vol.Schema({
+            vol.Optional(const.ATTR_INSTANCE_ID, default=1): vol.Coerce(int),
+        })
+        add_node_schema = vol.Schema({
+            vol.Optional(const.ATTR_INSTANCE_ID, default=1): vol.Coerce(int),
+            vol.Optional(const.ATTR_SECURE, default=False): vol.Coerce(bool),
+        })
+        node_schema = vol.Schema({
+            vol.Required(const.ATTR_NODE_ID): vol.Coerce(int),
+            vol.Optional(const.ATTR_INSTANCE_ID, default=1): vol.Coerce(int),
+        })
+        set_config_parameter_schema = vol.Schema({
```

Comment:
Can you skip making them variables, just pass `schema=vol.Schema`. That's the direction of HA.

Reply:
makes sense, thanks for explaining. will adjusty